tests/processing/strategies/test_streaming.py

```diff
@@ -320,11 +320,11 @@ def test_message_batch() -> None:
     smm = SharedMemoryManager()
     smm.start()
 
-    block = smm.SharedMemory(4096)
-    assert block.size == 4096
+    block = smm.SharedMemory(16384)
+    assert block.size == 16384
```

Comment:
I also considered changing the test such that it works with any page size, something like:
```
assert block.size == max(resource.getpagesize(), 4096)
```
However this seemed to be both more work and harder to reason about so I opted to change the numbers instead.